psalm.xml.dist

```diff
@@ -18,5 +18,6 @@
     </projectFiles>
     <plugins>
         <pluginClass class="Psalm\PhpUnitPlugin\Plugin"/>
+        <pluginClass class="Boesing\PsalmPluginStringf\Plugin"/>
```

Comment:
What is this? 🤔

Reply:
It parses `sprintf` and is able to understand if the return value is a `non-empty-string` for example.
But it also verifies that the amount of arguments passed to the `sprintf` do match with the amount of placeholders, etc.

http://github.com/boesing/psalm-plugin-stringf